requirements.txt

```diff
@@ -36,17 +38,22 @@ rdflib==4.2.1
 selenium==2.47.1
 coverage==3.7.1
 pyasn1==0.1.8
+pycparser==2.14
+cffi==1.2.1
```

Comment:
This is done, I've added `libffi-dev` to patches and snowball via salt and restarted the jenkins build.